src/options.h

```diff
@@ -22,6 +22,8 @@
 // Must be bigger than GC_MAX_SZCLASS.
 #define ARRAY_CACHE_ALIGN_THRESHOLD 2048
 
+#define DUMP_BACKTRACE_OF_STOP_THE_WORLD_STRAGGLER
```

Comment:
suggestion: make this time configurable, with a value of zero indicating to disable it
```suggestion
#define DUMP_BACKTRACE_OF_STOP_THE_WORLD_STRAGGLER (120 * 1000000) // 120 seconds
```

Reply:
Right. I think we might even want to make this some kind of runtime/CLI option.

Reply:
Yes, that would be preferable to adding a compiler ifdef here